src/config.c

```diff
@@ -2486,7 +2482,7 @@ standardConfig configs[] = {
     createIntConfig("rdb-key-save-delay", NULL, MODIFIABLE_CONFIG, INT_MIN, INT_MAX, server.rdb_key_save_delay, 0, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("key-load-delay", NULL, MODIFIABLE_CONFIG, INT_MIN, INT_MAX, server.key_load_delay, 0, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("active-expire-effort", NULL, MODIFIABLE_CONFIG, 1, 10, server.active_expire_effort, 1, INTEGER_CONFIG, NULL, NULL), /* From 1 to 10. */
-    createIntConfig("hz", NULL, MODIFIABLE_CONFIG, 0, INT_MAX, server.config_hz, CONFIG_DEFAULT_HZ, INTEGER_CONFIG, NULL, updateHZ),
+    createIntConfig("hz", NULL, MODIFIABLE_CONFIG, CONFIG_MIN_HZ, CONFIG_MAX_HZ, server.config_hz, CONFIG_DEFAULT_HZ, INTEGER_CONFIG, NULL, updateHZ),
```

Comment:
This is a backwards breaking change. Someone might have had a value of like 12, which was rounded up to CONFIG_MIN_HZ. With this change it will fail to startup. I would recommend keeping the code in updateHZ but having the range be 1, INT_MAX.

Reply:
I'm not sure what you mean, CONFIG_MIN_HZ is 1.
Whether to change it to 1 ~ 1000, maybe some users have set it to 500 ~ 1000.

Reply:
Oh, I see, I understand what you mean now.

Can we fix the bug so that the value behaves as intended, so that we actually set the bounds on startup?

Reply:
After reading through the code, I felt that it was too ugly to do it, and found that the best way was to use the code before #6624 instead.

Reply:
I suddenly realized where to put this code.